tcms/testcases/templates/testcases/mutable.html

```diff
@@ -211,7 +211,11 @@
                     <p class="help-block">{% trans "Email addresses separated by comma. A notification email will be sent to each Email address within CC list." %}</p> </div>
             </div>
 
-            <button type="submit" class="btn btn-default btn-lg">{% trans "Save" %}</button>
+            <div class="form-group">
+                <div class="col-md-1 col-lg-1">
+                    <button type="submit" class="btn btn-default btn-lg">{% trans "Save" %}</button>
+                </div>
+            </div>
```

Comment:
Good catch, but this needs to be consistent across pages. See also:

- testcases/clone.html, testcases/search.html
- testplans/mutable.html, testplans/clone.html, testplans/search.html
- testruns/mutable.html, testruns/search.html

Note: for search pages and for testruns/mutable.html there are other elements below the submit button so the problem is not so visible. Adding the surrounding divs will likely introduce a small change in the existing margins. We'll have to see how it looks but generally I prefer to have everything consistent across all templates.

Reply:
I agree. I think the most relevant page is new test plan. There the issue is not observed, I think it depends of the overall height of the form`s elements. If in the future new elements are added the issue will appear there too(this will happen for all pages that have button as last element). We have two options: 
1. Add margin-bottom to the body element. So it will be consistent across all pages;
2. Add new footer section, it will be just an empty div but in the future it may be useful. Right now there are two main section: the navbar and main content.

Of course we can leave it as it is.